src/scarlet2/questionnaire/questions.yaml

```diff
@@ -0,0 +1,91 @@
+initial_template: '{{code}}'
+questions:
+- question: Do you have a single image, or multiple images?
+  answers:
+  - answer: Single image
+    tooltip: A single image to model
+    templates:
+      - replacement: code
+        code: "channels=['g', 'r', 'i', 'z', 'y']\n\n{{obs}}"
+    followups: []
+    commentary: "We start with initializing the channels for the observation. The channels in the image can be specified as a list of the filters, like the above example."
+  - answer: Multiple images
+    tooltip: A single image to model
+    templates: []
+    followups:
+    - question: Do you wish to model transient or variable sources across multiple epochs?
+      answers:
+      - answer: "Yes"
+        templates:
+          - replacement: code
+            code: "channels=[(‘g’,0)]
+                   \nchannels=[(‘g’,1’)]\
+                   \n\n{{obs}}"
+        commentary: "For each image, the channels should be a list of tuples where each tuple provides the filter and an epoch identifier which is unique for each image of a particular filter. For example, you might have two g-band images from different times, which would be initialized separately with channels=[(‘g’,0)] and channels=[(‘g’,1’)]. You might have two other r-band images taken at different times again. These would be initialized with channels=[(‘r’,0)] and channels=[(‘r’,1’)]. You may wish to take note of any epochs where the transient is known to be ‘off’ - you can add this constraint later if desired. Each image can be initialized"
```

Comment:
Has this commentary been cut off at the end? Also, what is the single quote for in `channels=[('g',1']` and `channels=[('r',1')]`?